extra/utils/validation.js

```diff
@@ -0,0 +1,59 @@
+const users = [{
+    traineeEmail: 'trainee1@successive.tech',
+    reviewerEmail: 'reviewer1@successive.tech',
+    },
+    {
+        traineeEmail:'urvashi.chaudhary@successive.tech',
+        reviewerEmail:'mohammad.adil@successive.tech',
+    },
+    {
+        traineeEmail:'i@gmail.com',
+        reviewerEmail:'jfjjjjf',
+    },
+];
+
+
+function validateEmail(input){
+    let correct = /^[a-zA-Z0-9._-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,4}$/;
+    
+    return correct.test(input);
+}
+
+
+
+
+function validateUser(users){
+    let valid=0;
+    let invalid=0;
+    let v_users="";
```

Comment:
its better to take arrays to push valid users and invalid users instead of concatenating them in string.
also no need of a counter to track number of valid and invalid users
we can simply manage this with the arrays length.

Reply:
done